website/docs/components/form/text-input/partials/code/how-to-use.md

```diff
@@ -143,7 +148,7 @@ Pass an `@extraAriaDescribedBy` argument to the field to connect one or more ext
 
 #### HTML native attributes
 
-`...attributes` spreading is supported in the field. This means you can use all the standard HTML attributes of the `<input>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (eg. providing a `name` for the control, or adding `min` `max` `minlength` `maxlength` `pattern` attributes to it).
+This component supports use of `...attributes`. This means you can use all the standard HTML attributes of the `<input>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (e.g., providing a `name` for the control, or adding `min` `max` `minlength` `maxlength` `pattern` attributes to it).
```

Comment:
```suggestion
This component supports use of `...attributes`. This means you can use all the standard HTML attributes of the `<input>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (e.g., providing a `name` for the control, or adding `min`, `max`, `minlength`, `maxlength`, or `pattern` attributes to it).
```